src/StudioProjectLoader.tsx

```diff
@@ -171,4 +175,33 @@ export class StudioProjectLoader {
             console.error(`[${this.saveDocument.name}] There was an issue fetching the current document state`);
         }
     };
+
+    public onFetchOutputSettings = async (): Promise<UserInterfaceOutputSettings[] | null> => {
+        if (this.userInterfaceID) {
+            const outputSettings = await axios.get(`${this.graFxStudioEnvironmentApiBaseUrl}/output/settings`, {
+                headers: { Authorization: `Bearer ${this.authToken}` },
+            });
+            const userInterface = await axios.get(
```

Comment:
do we already have a ticket for Env API to implement it in one request

Reply:
We don't, because they will not, the new contract for the `user-interfaces/id` will include only the `intent` regarding the output settings, f.e `{id: 1, 1213234:{layoutIntent: ['print']}`so the second call to fetch all output settings and map will always be there.